components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/servlet/SAMLSSOProviderServlet.java

```diff
@@ -2122,4 +2166,24 @@ private void handleInvalidRequest(String msg, SAMLSSOAuthnReqDTO authnReqDTO, Ht
         sendNotification(errorResp, SAMLSSOConstants.Notification.EXCEPTION_STATUS, SAMLSSOConstants
                 .Notification.EXCEPTION_MESSAGE, authnReqDTO.getAssertionConsumerURL(), req, resp);
     }
+
+    /**
+     * This method is used to retrieve login tenant domain.
+     *
+     * @param req     HttpServletRequest.
```

Comment:
Won't it be nice if we have same indent in param and return description?

Reply:
Addressed with https://github.com/wso2-extensions/identity-inbound-auth-saml/pull/348